rosdep/python.yaml

```
@@ -3693,7 +3693,17 @@ python-pymavlink:
       packages: [pymavlink]
 python-pymodbus:
   debian: [python-pymodbus]
-  ubuntu: [python-pymodbus]
+  ubuntu:
+    '*': [python-pymodbus]
+    focal: null
+    jammy: null
```

Actually, future ubuntu releases will also not provide python 2 packages, so better use:

```suggestion
    bionic: [python-pymodbus]
```

melodic is currently the only ROS distribution supporting python2, and that uses Ubuntu Bionic.

---

Thanks for the heads-up.